templates/default/rt/rtticketadd.html

```diff
@@ -121,22 +121,15 @@ <H1>{$layout.pagetitle}</H1>
 			<IMG src="img/queue.gif" alt="">
 			{trans("Categories:")}
 		</TD>
-		<TD style="width: 99%;" class="nobr">
-			<TABLE width="100%" cellpadding="1" {tip trigger="categories"}>
-				{foreach $categories as $category}
-				<TR>
-					<TD style="width: 1%;" class="text-center">
-						<INPUT type="checkbox" name="ticket[categories][{$category.id}]" id="cat{$category.id}" value="1" {if $category.checked > 0}checked{/if}>
-					</TD>
-					<TD style="width: 99%;">
-						<label for="cat{$category.id}">{$category.name}</label>
-					</TD>
-				</TR>
-				{/foreach}
-			</TABLE>
+		<TD>
+		{foreach $categories as $category}
+		<span class="nobr">
+                <INPUT type="checkbox" name="ticket[categories][{$category.id}]" id="cat{$category.id}" value="1" {if $category.checked > 0}checked{/if}>
+                <label for="cat{$category.id}">{$category.name}</label>
```

Comment:
To jeśli już takie coś robisz to aż się prosi, żeby upraszczać wszystkie labele na:
```html
<label>
     <input ....>
     tekst
</label>
```

Reply:
Poszedł fixup.